react/Product.js

```diff
@@ -14,7 +14,15 @@ const getPrice = path(['commertialOffer', 'Price'])
 const getTax = path(['commertialOffer', 'Tax'])
 const getAvailableQuantity = pathOr(0, ['commertialOffer', 'AvailableQuantity'])
 
-const getFinalPrice = (value, getPriceFunc, { decimals, pricesWithTax }) => {
+const getFinalPrice = (
+  value,
+  getPriceFunc,
+  { decimals, pricesWithTax, pricesHidden }
+) => {
+  if (pricesHidden) {
+    return 0
```

Comment:
I think google will mislead a 0 price with a free offer. For this situation I think the solution is to [remove the Offer (since is an optional property)](https://moz.com/community/q/post/320468) and not set the price as 0. What do you think?